### PR TITLE
#6 Add in first draft for engineering principles

### LINKED ISSUE #6: Add engineering strategy

### PR DESCRIPTION
**Pull request by pvhee**
Fixes #6

### CODE REVIEW

**Comment by pvhee** [comment]
@ayanozturk @adamclark-dev @NoelLH would appreciate your feedback on the above

**Review by adamclark-dev — APPROVED** [review]

File: engineering-principles.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+* Real-time data and monitoring
+* [Accessibility](frontend/accessibility.md) at the heart of everything
+* Going beyond the screen
+
+We approach those principles in the following manner:
+
+* Development of services over maintaining infrastructure and manual scaling. Functions over microservices, microservices over monoliths, managed services over self-managed services. [12-factor apps](https://12factor.net/) where applicable
+* Use of and contribute to open source software. [Coding in the open as our default setting](open-source.md).
+* Continuous delivery backed up by automated testing. Deploy to production systems multiple times a day.
+* Real-time monitoring of live products through dashboards (e.g. Grafana), logs (e.g. Loggly, CloudWatch) and alerts (e.g. Wormly, Slack)
```
Comment (NoelLH) [inline engineering-principles.md]: Slack's just a channel for monitors to ping to?

**Review by NoelLH — APPROVED** [review]
One tiny thing inline @pvhee, but all looks good to me

File: engineering-principles.md (hunk 1 above)
Comment (pvhee) [reply engineering-principles.md]: True, have taken off Slack, there's no need to list it here.

**Pull request merged**